t/t9392-python-callback.sh

```diff
@@ -64,9 +64,9 @@ test_expect_success '--message-callback' '
 	setup message-callback &&
 	(
 		cd message-callback &&
-		git filter-repo --message-callback "return b\"TLDR: \"+message[0:5]" &&
+		git filter-repo --message-callback "return b\"TLDR: \"+message[0:5]+b\" \"+os.environ[\"GIT_COMMIT\"].encode(\"utf-8\")" &&
```

Comment:
Without the change to _tweak_commit() that you made, you could just use a commit callback in order to have access to all information about the commit, and make the same kind of tweak to the commit message:

git filter-repo --commit-callback "commit.message = b\"TLDR: \"+commit.message[0:5]+b\" \"+commit.original_id"

Reply:
Aha! Since I was only intending on changing the message, I had sorta tunneled on the `--message-callback` section of the docs, failing to see the `--commit-callback` - yep this makes sense!

I wanted the message callback to have access to the old commit id - so that we could leave a trace of the old commit ID in the rewritten message. It appears that `-message-callback` isn't good for this, but `--commit-callback` is!

I may be able to propose a piece of documentation that may make this more obvious (to me), but for now I'll close this!